content/company-info-and-process/onboarding/product-onboarding-videos.md

```diff
@@ -40,6 +40,6 @@ We created these videos as an intro to educate the team on our product. The goal
 
 <div style="position: relative; padding-bottom: 66.42066420664207%; height: 0;"><iframe src="https://www.loom.com/embed/3da7f39adae94ad1ae4a8516ca154328" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
 
-## Cloud SaaS Video
+## IAM & Admin Exp Video
```

Comment:
If we are starting a poll here, I vote for "IAM and Admin Exp" 😁 Partly because having "&" in titles and headings feels somewhat, unofficial... also being consistent that not all places are supporting "&" to be part of.

Reply:
@miveronese - Can you remove the Cloud SaaS video and add a note that we will re-record this video once we are aligned on mission/vision/roadmap?